packages/url_launcher/url_launcher/example/linux/flutter/generated_plugin_registrant.cc

```diff
@@ -8,7 +8,6 @@
 
 void fl_register_plugins(FlPluginRegistry* registry) {
   g_autoptr(FlPluginRegistrar) url_launcher_linux_registrar =
-      fl_plugin_registry_get_registrar_for_plugin(registry,
-                                                  "UrlLauncherPlugin");
+      fl_plugin_registry_get_registrar_for_plugin(registry, "UrlLauncherPlugin");
```

Comment:
This file needs a revert. (Re-fixing this in Flutter so it stops happening is waiting to land.)